secure-enclave/templates/setup-app-deployment.yaml

```diff
@@ -25,6 +25,7 @@ spec:
       {{- end }}
       containers:
       - name: {{ .Values.setupApp.image.repository }}
+        imagePullPolicy: {{ .Values.setupApp.image.pullPolicy }}
         image: {{ .Values.setupApp.image.registry }}/{{ .Values.setupApp.image.repository }}:{{ .Values.setupApp.image.tag }}
         {{- range .Values.setupApp.service.ports }}  
```

Comment:
I think this needs to be port (singular) instead of ports (plural) to match setup-app-svc.yaml:12